locker-dome/locker-dome-htb.js

```diff
@@ -174,6 +174,10 @@ function LockerDomeHtb(configs) {
                 applies: gdprStatus.applies,
                 consent: gdprStatus.consentString
             };
+            var uspStatus = ComplianceService.usp.getConsent();
+            payload.us_privacy = {
```

Comment:
Our lint rules require the use of camel case. Since this property is being used to signal consent information to your endpoint, you can disable the lint rule for this line. You can verify there are no additional lint errors by running `npm run lint` from the root of your locker-dome directory.

Reply:
Thank you, this has been adjusted. I also added another indentation to the line below to conform w/ the rest of the code.